### ISSUE TITLE
negative mouse coordinates returned as 2^64

### ISSUE THREAD

**Issue by danyshaanan** [octalmage/robotjs#15]
When using multiple screens, zero is defined as top left of the main screen, which means that everything to it's left/top should have negative x/y coordinates accordingly. However, those are returned as 2^64.

This image shows the output of the script below as I move the mouse to the left-most screen, and then to the top of it. (`node index.js >> data.txt` and `tail -f data.txt`)

Could this simply be a problem with integers becoming unsigned in the C code?

![robotjsnegativemouse](https://cloud.githubusercontent.com/assets/1116738/6347188/984e35b4-bc1e-11e4-95a2-2e14dd59eccc.png)

```js
'use strict'

var robot = require('robotjs')

function encodeMouse(mouse) {
  return '[' + mouse.x + ',' + mouse.y + ']'
}

var mouse
var prevCode

setInterval(function() {
  var code = encodeMouse(robot.getMousePos())
  if (code !== prevCode) {
    console.log(code)
  }
  prevCode = code
}, 100)
```

**Comment by octalmage** [comment]
Good find! I actually never tested in a multi-monitor set up. I'll check it out and let you know what I figure out!

**Comment by ryanramage** [comment]
yup, I see this as well. 

On one external monitor:

> Mouse is at x:18446744073709550000 y:18446744073709552000

and on the other external monitor:

> Mouse is at x:1171 y:18446744073709552000

**Comment by OverZealous** [comment]
I think related to this, I'm getting a `SEGFAULT` trying to extract the color from a far-left monitor (using the example code).

```
$ node index.js
#e8e8e8 at x:18446744073709552000 y:310
Segmentation fault: 11
```

**Comment by Deltatiger** [comment]
Seems js doesn't have signed and unsigned numbers. Converting all numbers to a uniform 64 bit format will help I guess. Will have to check it out

**Comment by octalmage** [comment]
This has been fixed in #48 which has been merged. I'll publish to npm later today.

**Comment by danyshaanan** [comment]
Thanks! :]
